### PR TITLE
jQuery 3.5.1

### PR DESCRIPTION
**Pull request by agriffard**
http://blog.jquery.com/2020/04/10/jquery-3-5-0-released/

Didn't change it in the themes for the moment, as their jQuery files are in a vendor folder.

### CODE REVIEW

**Comment by sebastienros** [comment]
Let's give it a week and see if there is a 3.5.1 ;)

**Comment by agriffard** [comment]
https://www.infoq.com/news/2020/04/jquery-35-xss-vulnerability-fix/

**Comment by agriffard** [comment]
Updated to 3.5.1
http://blog.jquery.com/2020/05/04/jquery-3-5-1-released-fixing-a-regression/

**Comment by agriffard** [comment]
The Media library does not seem to work anymore with jQuery 3.5.1.

Anyone can help on this one?

**Comment by jptissot** [comment]
I will try to debug this when I get a chance.

**Comment by agriffard** [comment]
Thank you @jptissot .

/cc @Skrypt Can you please also have a look if you have time?

**Comment by jptissot** [comment]
Found the issue, pushing...

**Comment by jptissot** [comment]
@agriffard are we using npm or yarn ? There is a yarn.lock file and a package-lock.json file in the repo. We should choose one and stick with it.

File: src/OrchardCore.Modules/OrchardCore.Setup/package.json [hunk 1]
```diff
@@ -4,6 +4,6 @@
   "dependencies": {
     "bootstrap": "4.4.1",
     "popper.js": "1.16.0",
-    "jquery": "3.4.1"
+    "jquery": "3.5.0"
```
Comment (jptissot) [inline src/OrchardCore.Modules/OrchardCore.Setup/package.json:7]: Maybe 3.5.1 here too

File: src/docs/reference/modules/Resources/README.md [hunk 2]
```diff
@@ -130,10 +130,10 @@ settings.AtFoot();
 ##### Set the version to use
 
 ```csharp
-settings.UseVersion("3.3");
+settings.UseVersion("3.4");
```
Comment (jptissot) [inline src/docs/reference/modules/Resources/README.md:133]: ?

**Comment by agriffard** [comment]
> are we using npm or yarn ?

npm should be used.

**Comment by jptissot** [comment]
Pushing one last commit.

File: src/OrchardCore.Modules/OrchardCore.Setup/package.json (hunk 1 above)
Comment (jptissot) [reply src/OrchardCore.Modules/OrchardCore.Setup/package.json:7]: Fixed

**Pull request merged**